stdlib/src/lib.rs

```diff
@@ -51,6 +51,112 @@ impl Default for StdLibrary {
     }
 }
 
+// FALCON SIGNATURE
+// ================================================================================================
+
+/// Event ID for pushing a Falcon signature to the advice stack.
+/// This event is used for testing purposes only.
+pub const EVENT_FALCON_SIG_TO_STACK: u32 = 3419226139;
+
+/// Gets as input a vector containing a secret key, and a word representing a message and outputs a
+/// vector of values to be pushed onto the advice stack.
```

Comment:
nit: I'd update this to be roughly along the lines mentioned in https://github.com/0xPolygonMiden/miden-vm/pull/1703#discussion_r2002137095.

Reply:
Ah, I think I had misunderstood. Removed the `Return` section, and moved up the sentence from there, i.e. this is now

```
/// Signs the provided message with the provided secret key and returns the resulting signature
/// encoded in the format required by the rpo_faclcon512::verify procedure, or `None` if the secret
/// key is malformed due to either incorrect length or failed decoding.
```